ote/src/cljs/ote/ui/spinner.cljs

```diff
@@ -73,6 +73,7 @@
           (merge
             (when-let [style (:style options)]
```

Comment:
Alternative implementation
```suggestion
           (:style options)
```
Because when returns a nil if style isn't defined, so does the suggestion. FYI @jkorpelin

Reply:
But as commented in an another part: shouldn't the material-ui spinner be used instead of this...

Reply:
I change it to: (when (:style options) (:style options))
Because no need to merge when options is null.